app/controllers/home_controller.rb

```diff
@@ -19,9 +19,15 @@ def list
   end
 
   def download
-    send_data Evaluation.to_csv(params.to_json), {
+    send_file Rails.root.join('lib', 'data', 'seed', 'PAME_Data-2018-07-11.csv'), {
               type: "text/csv; charset=iso-8859-1; header=present",
               disposition: "attachment",
               filename: "protectedplanet-pame.csv" }
+    # This needs to be uncommented once we have sorted out the CSV generation issue
```

Comment:
Does the CSV for the download need to look different from the CSV we use for the import? @drewpotter @rogeringle